lib/webbluetooth/bindings.js

```diff
@@ -0,0 +1,267 @@
+var events = require('events');
+var util = require('util');
+
+var debug = require('debug')('bindings');
+
+var ble = navigator.bluetooth;
+
+//stub data until web APIs are implemented.
+var tempResponses = require('./tempResponses');
+
+
+function addDashes(uuid){
+  if(uuid && uuid.length === 32){
+    uuid = uuid.substring(0,8) + '-' + uuid.substring(8,12) + '-' +  uuid.substring(12,16) + '-' +  uuid.substring(16,20) + '-' +  uuid.substring(20);
+  }
+  return uuid;
+}
+
+function stripDashes(uuid){
+  if(uuid){
+    uuid = uuid.split('-').join('');
+  }
+  return uuid;
+}
+
+
+var NobleBindings = function() {
+
+  this._startScanCommand = null;
+  this._peripherals = {};
+
+  var self = this;
+
+  setTimeout(function(){
+    self.emit('stateChange', 'poweredOn');
+  }, 50); //maybe just a next tick?
```

Comment:
There are none for now but we're talking about it at https://github.com/WebBluetoothCG/web-bluetooth/issues/127